runner/database.go

```diff
@@ -10,14 +10,13 @@ import (
 	"strconv"
 	"strings"
 
-	"github.com/multiprocessio/go-json"
-
 	_ "github.com/ClickHouse/clickhouse-go/v2"
 	_ "github.com/denisenkom/go-mssqldb"
 	_ "github.com/go-sql-driver/mysql"
 	"github.com/jmoiron/sqlx"
 	_ "github.com/lib/pq"
 	_ "github.com/mattn/go-sqlite3"
+	jsonutil "github.com/multiprocessio/go-json"
```

Comment:
This shouldn't change here. I keep internal projects separate from external projects. Since go-json is a multiprocess library it stays separate